addon/package.json

```diff
@@ -2,7 +2,7 @@
   "title": "Test Pilot",
   "name": "testpilot-addon",
   "id": "@testpilot-addon",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
@lmorchard i bumped the version in three places which seems not good, but maybe that's how it's done